dagshub/streaming/mount.py

```diff
@@ -91,13 +113,15 @@ def mount(debug=False,
           repo_url: Optional[str] = None,
           branch: Optional[str] = None,
           username: Optional[str] = None,
-          password: Optional[str] = None):
+          password: Optional[str] = None,
+          token: Optional[str] = None):
     logging.basicConfig(level=logging.DEBUG)
     fuse = DagsHubFUSE(project_root=project_root, repo_url=repo_url, branch=branch, username=username,
-                       password=password)
-    print(
+                       password=password, token=token)
+    logger.debug(
         f'Mounting DagsHubFUSE filesystem at {fuse.fs.project_root}\n'
         f'Run `cd .` in any existing terminals to utilize mounted FS.')
```

Comment:
This should be either a print or a warning. User must see this

Reply:
Sorry, aggressive renaming.